go/vt/vtgate/evalengine/abs.go

```diff
@@ -0,0 +1,52 @@
+package evalengine
```

Comment:
I don't think `abs` needs to have its own file. You can put this implementation if `func.go`.

Reply:
I've solved it